### PR TITLE
Create integrate-carbon-deamon-with-rs-metrics

### PR DESCRIPTION
**Pull request by kylelaffoon**

### CODE REVIEW

**Comment by kylelaffoon** [comment]
@kmsholcomb Please edit at your convenience.

**Comment by kmsholcomb** [comment]
Hi @kylelaffoon. See the attached Word doc for some suggested edits. Thanks. 

[Integrate Carbon Daemons with Rackspace Metrics_editsKH.docx](https://github.com/rackerlabs/rackspace-how-to/files/159316/Integrate.Carbon.Daemons.with.Rackspace.Metrics_editsKH.docx)

File: content/rackspace-metrics/integrate-carbon-deamon-with-rackspace-metrics.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+#### Run Carbon Forwarder
+
+Use the following command to run Carbon Forwarder.
+
+    twistd blueflood-forward
+
+| **Switch** | **Description** | **default** |
+| ---------- | --------------- | ----------- |
+| -e | Endpoint to listen on for pickle protocol metrics | tcp:2004 |
+| -i | Metrics send interval, in seconds | 30.0 |
```
Comment (wolfdancer) [inline content/rackspace-metrics/integrate-carbon-deamon-with-rackspace-metrics.md]: @kylelaffoon this line is correct.  Thanks for changing "sec" to "in seconds".

**Comment by wolfdancer** [comment]
@kylelaffoon, the dependencies are all standard pip packages.   

twistd - https://pypi.python.org/pypi/Twisted
mock - https://pypi.python.org/pypi/mock
pytest - https://pypi.python.org/pypi/pytest
txKeystone - https://pypi.python.org/pypi/txKeystone

FYI - I did a search to see how dependencies are specified in python community. It seems that laying out dependency is not a standard practice.  I'll find a python expert and ask for opinion. I might file a RP based on what I learn later.

So far this content looks good to publish.

**Comment by kylelaffoon** [comment]
Thank you for the links, @wolfdancer . 

I have incorporated those into this PR. I will request this be merged while you are researching, and will make any further changes based on your later PR, as needed.

**Pull request merged**